config/config.go

```diff
@@ -1138,8 +1138,7 @@ func SetupViper(v *viper.Viper, filename string, bidderInfos BidderInfos) {
 	v.SetDefault("default_request.type", "")
 	v.SetDefault("default_request.file.name", "")
 	v.SetDefault("default_request.alias_info", false)
-	v.SetDefault("blacklisted_apps", []string{""})
-	v.SetDefault("blacklisted_accts", []string{""})
```

Comment:
It looks like the concept of blacklisted accounts was removed a while ago but this `SetDefault` call was missed, is that correct?

Reply:
Correct.